scripts/export_requirements.py

```diff
@@ -19,10 +19,12 @@
 
 
 GENERATED_FILE = pathlib.Path("requirements.txt")
+DOC_REQUIREMENTS = pathlib.Path("docs/.requirements.txt")
 
 VERSION_RESTRICTER_REGEX = re.compile(r"(?P<sign>[<>=!]{1,2})(?P<version>\d+\.\d+?)(?P<patch>\.\d+?|\.\*)?")
 PLATFORM_MARKERS_REGEX = re.compile(r'sys_platform\s?==\s?"(?P<platform>\w+)"')
 
+PACKAGE_REGEX = re.compile(r"^(?P<package>[^=<>~]+)")
```

Comment:
This can be changed to just match `==` as we never edit the file manually and it only gets edited by this script, where it only uses `==`.

Reply:
This comment can be ignored, see: https://discord.com/channels/798235512208490526/845824657306288158/909740574322610177